frameworks/Python/web2py/setup.sh

```diff
@@ -0,0 +1,4 @@
+export PY2_ROOT=$IROOT/py2
+export PY2=$PY2_ROOT/bin/python
+
+sudo chmod a+w web2py/httpserver.pid & $PY2 web2py/web2py.py -a '' -i 127.0.0.1 -p 8080  &
```

Comment:
I tried running the tests without that and it seems to work fine, so I'll remove the sudo command. I think it was a remnant of when I was still learning web2py and was having trouble with some of their boilerplate files.